objectherkenning_openbare_ruimte/databricks_pipelines/post_processing_pipeline/data_enrichment_step/components/decos_data_connector.py

```diff
@@ -191,8 +191,14 @@ def add_permit_coordinates_columns(self, df, addresses):
         for address in addresses:
             coordinates = self.convert_address_to_coordinates(address)
             if coordinates:
-                longitudes.append(coordinates[0])
-                latitudes.append(coordinates[1])
+                if coordinates[0] > coordinates[1]:
```

Comment:
I would suggest we actually update [lines 174/175](https://github.com/Computer-Vision-Team-Amsterdam/Objectherkenning-Openbare-Ruimte/blob/3e6906e8c19fead1252371b36bdfeb52760124bb/objectherkenning_openbare_ruimte/databricks_pipelines/post_processing_pipeline/data_enrichment_step/components/decos_data_connector.py#L174) to match the current API, which is `[lat, lon]` instead of `[lon, lat]`.

It's still a good idea to keep a check to see if the returned coordinates are roughly in the Amsterdam area, and if not throw an exception or something so the step actually fails.

What do you think?